docs/_content/ontwikkelaars/algemeen/notificaties.md

```diff
@@ -100,3 +127,163 @@ Bronnen:
 * [Digilevering koppelvlakspecificatie](https://www.logius.nl/sites/default/files/public/bestanden/diensten/DigiLevering/Koppelvlakspecificatie.pdf)
 * [Digilevering documentatie](https://www.logius.nl/diensten/digilevering/documentatie)
 * [Regie- en Zaakservices specificaties](https://www.gemmaonline.nl/images/gemmaonline/c/cd/Koppelvlakspecificatie_Regie-zaak_services_v1.0.pdf)
+
+# API specificatie (concept)
+
+[Voorbeeld sequence diagram (in-depth)](http://sequencediagram.org/index.html#initialData=C4S2BsFMAIDkHtQDMQGNICdIDsBQuAHAQw1FRGO2GgC0AlAYUJLIqKrkRBVSNBgbwAtgXjYcwZqTRsOdIgCMFYALIBFKa0rUAIo3z0GAWgB8CZGj4gBw0eKoAuaAAUA8gGUAKtAD0RAiA+AG4AjD4KGGL2ADrYPj4ADADMAEwhRgBeRADWOPG45tyW-NCCImISpvJKqmpODFh8MMDwAaix8clpmTl5PgVcPFY25fbAADxGRtXKwOpOAJI6TiEpSbiGk0aFQyVldhJOABKens7QKQkhHT4AQpHY0ADmmJBPIADOwFiYACbQQkg1CWKzW+VwAF4IQtsGAQERwJ9htAguxoAAqdGGTHQIhIF4PX5Q-B6YxmQbFaylWwVRwuDzePwBYJhRRRSCAqg4G5ddJZXJxfo7SkjA5UKqKWbzaDuACuCg+qAwIAUMFJ0BaGtaaB5qT5vUFAwsvD2NLGWxmtScrgA0rhSVthSaqftacBjqdzpdroKAIIKdmc6hEdgvIQh7LAcFQmFwhFIkqox6Y0k4vEEsREiEGRimJ3I11jJxuLy+fyBULhB72HyrJI+bAU52QD43W6YNAAC0k+dNo0qJktczqLkiv1l6ABLY+RBeWugIOgdaNRWb1P7VAtkqt0FtG0YjqbBbNh2gJzOFwSCRuADV4OAWvAQL98kP1KYTKTiwyy8zK43jWGD4fDEYAkxebA2w7VBu3yIA)
+
+### Abonneren
+
+Een consumer abonneert zich op notificaties door de volgende request naar de
+NC te sturen:
+
+```http
+POST /api/v1/subscribers
+
+Authorization: Bearer abcdef1234
+Content-Type: application/json
+
+{
+    "callbackUrl": "https://ref.tst.vng.cloud/zrc/api/v1/callbacks",
+    "auth": "Bearer aef34gh...",
+    "kanalen": [
+        "zaken",
+        "informatieobjecten"
+    ]
+}
+```
+
+Het `DELETE`n, `PATCH` en `PUT` op individuele subscribers moet mogelijk zijn.
+
+Details moeten ook opgevraagd kunnen worden, met uitzondering van de `auth`
+gegevens.
+
+Filter parameter(s) op `kanelen` zijn handig.
+
+### Kanalen beheren
+
+Een kanaal is het equivalent van een exchange in RabbitMQ.
+
+Kanalen worden aangemaakt door producers (bronnen), en zijn gekarakteriseerd
+door een unieke naam.
+
+```http
+POST /api/v1/kanalen
+
+Authorization: Bearer abcdef1234
+Content-Type: application/json
+
+{
+    "naam": "zaken"
+}
+```
+
+Een kanaal mag slechts eenmalig aangemaakt worden.
+
+Daarnaast moet het oplijsten van kanalen mogelijk zijn, zodat consumers kunnen
+zien waarop ze zich kunnen abonneren.
+
+### Notificaties insturen
+
+De NC moet notificaties kunnen ontvangen:
+
+```http
+POST /api/v1/notificaties
+
+Authorization: Bearer abcdef1234
+Content-Type: application/json
+
+{
+    "bronUrl": "https://ref.tst.vng.cloud/zrc/api/v1/zaken/d7a22",
```

Comment:
ik mis het kanaal?

```suggestion
    "kanaal": "zaken",
    "bronUrl": "https://ref.tst.vng.cloud/zrc/api/v1/zaken/d7a22",
```

Reply:
Klopt, was in vorige versie geen probleem omdat die naar `/bronnen/123/notificaties` gestuurd werd, is nu niet langer zo.